server/app.js

```diff
@@ -0,0 +1,44 @@
+const createError = require("http-errors");
+const express = require("express");
+const path = require("path");
+const cookieParser = require("cookie-parser");
+const logger = require("morgan");
+
+const controllers = require("./controllers");
+
+const app = express();
+
+app.use(logger("dev"));
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+app.use(cookieParser());
+
+app.use(controllers);
+
+/**
+ * Uncomment these lines after setting up react server
+ * 
+ * app.use(express.static(path.join(__dirname,'..', 'client', 'build')));
+ * app.get('/', (req, res) => {
```

Comment:
should be to catch all the requests with react-router
`app.get('*', (req, res) => {`

https://tylermcginnis.com/react-router-cannot-get-url-refresh/